roles/addons/slurm/tasks/main.yml

```
@@ -67,7 +60,7 @@
     home: "{{ slurm_home_path }}"
     state: present
 
-- name: Template >> slurm.conf
+- name: template █ Generate /etc/slurm/slurm.conf
```

Use `slurm_home_path` as in `dest`?
```suggestion
- name: "template █ Generate {{ slurm_home_path }}/slurm.conf"
```

---

Yes. Done